Plugins/Flow.Launcher.Plugin.Explorer/Search/Everything/EverythingAPI.cs

```diff
@@ -132,10 +146,12 @@ public static async ValueTask<bool> IsEverythingRunningAsync(CancellationToken t
 
                     var result = new SearchResult
                     {
+                        // todo the types are wrong. Everything expects uint everywhere, but we send int just above. pls fix
                         FullPath = buffer.ToString(),
                         Type = EverythingApiDllImport.Everything_IsFolderResult(idx) ? ResultType.Folder :
                             EverythingApiDllImport.Everything_IsFileResult(idx) ? ResultType.File :
-                            ResultType.Volume
+                            ResultType.Volume,
+                        Score = (int)EverythingApiDllImport.Everything_GetResultRunCount( (uint)idx) 
```

Comment:
I am wondering whether we should give score for other sorting (i.e. `-idx * c` for some constant)

Reply:
sorry, i don't understand what you're saying

Reply:
Feedback copied from discord.

Your changes look like this:

- In Everything API: if we use RunCount, then we use the score from RunCount times 5. if we dont use RunCount, then the score remains 0 (0 * 5 = 0)
so if i would have an entry with RunCount 1000, then the score would be 5000, and if i would have an entry with RunCount 0, then score would be 0
- in Search Manager, we ignore the existing score completely, and instead we simply set a score between [-49, +50], based on the position of the items in the array received from Everything API) (`Score = -i + 50`)

this second part looks like a brain fart (maybe you wanted `Score = Score + (-i + 50)`), but even ignoring that, i am not convinced it's the right way to sort. Using my RunCount 1000 example, this particular scoring would not really fix or change anything in the ordering of the items..

Right now, my suggestion is to remove the commit

Reply:
commit reverted